src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.pow(side, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: "
```

Comment:
The method `draw` is combining two different responsibilities: calculating the area and printing the square information. Consider separating the logic for calculating the area from the logic for printing to follow the Single Responsibility Principle.